src/components/ClusterTemplateDetailsPage/ClusterTemplateDetailsPage.tsx

```diff
@@ -38,7 +38,6 @@ import { getResourceVendor } from '../../utils/clusterTemplateDataUtils';
 import useClusterTemplateActions from '../../hooks/useClusterTemplateActions';
 import useClusterTemplateDeserializer from '../../hooks/useClusterTemplateDeserializer';
 import Alerts from '../../alerts/Alerts';
-import WithClusterTemplateQuickStarts from '../ClusterTemplatesGettingStarted/WithClusterTemplateQuickStarts';
```

Comment:
`WithClusterTemplateQuickStarts` can be removed completely, right ?

Reply:
I use it to generate this component in isolation in storybook
Perhaps I can put it in the story, or in a folder storybookUtils

Reply:
right. deleted.